config.py

```diff
@@ -1421,8 +1421,106 @@ class ArbitraryContrast(TypedDict):
 Whether to subtract the evoked signal (averaged across all epochs) from the
 epochs before passing them to time-frequency analysis. Set this to `True` to
 highlight induced activity.
+
+Note: Note
+    This also applies to CSP analysis.
+"""
+
+###############################################################################
+# TIME-FREQUENCY CSP
+# ------------------
+
+decoding_csp: bool = False
+"""
+Whether to run decoding via Common Spatial Patterns (CSP) analysis on the
+time-frequency representation of the data.
```

Comment:
you don't feed the TFR data as input.
```suggestion
Whether to run decoding via Common Spatial Patterns (CSP) analysis on the
data. CSP takes as input data covariances that are estimated on different
time and frequency ranges. This allows to obtain decoding scores defined over
time and frequency.
```